fuzz/client.c

```diff
@@ -84,9 +92,12 @@ int FuzzerTestOneInput(const uint8_t *buf, size_t len)
             }
         }
     }
+ end:
     SSL_free(client);
     ERR_clear_error();
     SSL_CTX_free(ctx);
+    BIO_free(in);
+    BIO_free(out);
```

Comment:
This is wrong as the ownership of the bios is transferred to the SSL object with the SSL_set_bio() above. Just free the bios in the unable to create BIO `if`.